### PR TITLE
[21834] - Project browser fix display errors for missing behaviors, add alert

### PR DESCRIPTION
**Pull request by BerndN**
unresolvable behaviors are now flagged with red, tooltip indicates "missing" clicking on graphic triggers alert

### CODE REVIEW

File: Toolset/palettes/project browser/behaviors/revideprojectbrowsercontrolrowbehavior.livecodescript [hunk 1]
```diff
@@ -222,7 +253,6 @@ setProp dvHilite[pHiliteColor] pBoolean
          set the backgroundColor of field "scriptLines" of me to revIDEColor("dataView_hiliteColor")
       end if
       
-     local tCPName
       repeat with i = 1 to 10
          put ("cBehaviorLongID" & i) into tCPName
          if exists(the tCPName of me) and the scriptStatus of the tCPName of me is "error" then
```
Comment (BerndN) [inline Toolset/palettes/project browser/behaviors/revideprojectbrowsercontrolrowbehavior.livecodescript:256]: missing setting the backgroundColor to red

**Comment by BerndN** [comment]
Sorry for the mess I created by the corrections. But the scripts now compile and work as expected.

**Comment by BerndN** [comment]
this is what it should look like

pb missing behavs

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok abf9105

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok abf9105d4661ef75e19f60a8d222fc091f911b04

**Pull request merged**